arangod/Graph/Enumerators/OneSidedEnumerator.cpp

```diff
@@ -121,11 +121,11 @@ auto OneSidedEnumerator<Configuration>::computeNeighbourhoodOfNextVertex()
 
     // Will throw all network errors here
     std::vector<Step*> preparedEnds = std::move(futureEnds.get());
-
     TRI_ASSERT(preparedEnds.size() != 0);
     TRI_ASSERT(_queue.firstIsVertexFetched());
   }
 
+  TRI_ASSERT(!_queue.isEmpty());
```

Comment:
This is not required to be here I guess, as we're checking this already in Line 116. But it does not harm.

Reply:
Done